safe_cli/operators/utils/safe_to_l2_migration.py

```diff
@@ -0,0 +1,165 @@
+safe_to_l2_migration = {
+    "_format": "hh-sol-artifact-1",
+    "contractName": "SafeToL2Migration",
```

Comment:
Maybe can be better move this file to a contracts folder, utils is very generic.

Reply:
Yes, I'm thinking that this might belong to safe-eth-py